docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+    trusted-output:
+        container_name: trusted-output
+        # TODO Probably don't need clerk stuff in here
+        environment:
+            - CI=$CI
+            - NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=$NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY
+            - CLERK_SECRET_KEY=$CLERK_SECRET_KEY
+            - E2E_CLERK_USER_USERNAME=$E2E_CLERK_USER_USERNAME
+            - E2E_CLERK_USER_PASSWORD=$E2E_CLERK_USER_PASSWORD
+        build:
+            context: .
+            dockerfile: Dockerfile
+        volumes:
+            - ./:/app/
+            - node_modules:/app/node_modules
+        restart: always
+        networks:
+            - trusted-output
+        ports:
+            - 3000:3000
+        healthcheck:
```

Comment:
it could - does it make a big difference on whether it lives here or there? I find it a bit more readable in this syntax compared to the dockerfiles:

```HEALTHCHECK --interval=30s --timeout=10s --retries=3 CMD curl -f http://localhost:8080/health || exit 1```

but maybe its just a preference thing?

Reply:
Yeah that syntax isn't very readable but it does have one advantage... I think that when it's deployed, AWS will use the health check to know when to restart the container.

Reply:
got it! moved to dockerfile 😄